lib/vrt/map.rb

```diff
@@ -54,7 +58,6 @@ def construct_lineage(string, max_depth)
 
     def walk_node_tree(string, max_depth: 'variant')
       id_tokens = string.split('.').map(&:to_sym)
-      return nil if id_tokens.size > 3
```

Comment:
This was basically a double verification of the id, and is definetly OOS of what this function is doing, that's why I removed it

Reply:
Yep, valid!